### PR TITLE
Re-organizing and Updating 

### LINKED ISSUE #55: preprocess data that enters app
The data entering the app has an old preprocessing technique. It is necessary to import the proper preprocessing technique from a different notebook/script

### PR DESCRIPTION
**Pull request by antter**
I made a lot of updates to different parts of the system, and now I am making sure they all work neatly together. Everything is almost done, I just need to fix some pre-commit issues and test things out to make sure it all works

EDIT: Fixes #55

### CODE REVIEW

File: models/train_models.ipynb [hunk 1]
```diff
@@ -27,7 +27,7 @@
   },
```
Comment (aakankshaduggal) [inline models/train_models.ipynb:27]: <p>I am confused why the index looks 0 for all ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/github-labeler/pull/64/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Comment (antter) [reply models/train_models.ipynb:27]: <p>Ah this is because I make a small dataframe at each iteration then concatenate them, should I fix this?</p>

Comment (aakankshaduggal) [reply models/train_models.ipynb:27]: <p>Looks good, Thank you :) </p>

**Review by aakankshaduggal — COMMENTED** [review]
/lgtm 
@antter Can you please squash the commits and I can merge it :)

**Pull request merged**